lib/server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const express = require('express');
+require('dotenv').config();
+const app = express();
+const timestamp = require('./middleware/timestamp.js');
```

Comment:
I didn't catch this yesterday, but this needs to be `'./middleware/timestamps.js'`